docs/security/user-certs.md

```diff
@@ -25,36 +35,23 @@ Your **name** and **email** address are encoded into the signed certificate and
+        certificates using an OpenID provider.
 
 1. Click the `Log On` button and enter your instutional credentials.
 1. Upon successfully entering your credentials, you'll get a page asking for you to enter a password.  Enter a password that is at least 12 characters long and then click on the `Get New Certificate` button.<br>![Password entry](/img/cilogon_cert_password.png) 
 1. The web page will generate a `usercred.p12` file and prompt you to download it.  The certiticate will be protected
 using the password you entered in the prior step.
 
-### Getting a Certificate from the OSG CA
-
-There are two different ways to obtain your certificate, either via a command line interface, or through a web browser.
-The links below will guide you through whichever method you choose.
-The command line method is generally simpler and less prone to errors than using a web browser but it may require extra
-setup and package installation the first time you do it.
-Most people end up needing their certificate for both command line grid use and web browser use and it is generally
-easier to import your certificate into your browser (and email client) from the command line certificate files than to
-export your certificate from your web browser to use on the command line. 
-This is because of the large variety of web browsers each with its own way to deal with certificates. If you will use
-the certificate ONLY in your web browser then requesting it from your browser is probably the easiest method.
-
-[Get or renew a certificate with command line interface.](/security/certificate-management.md)
-
 ### PKCS12 (.p12) vs PEM format
 
-Your certificate and key pair can be stored as separate files, by default named `~/.globus/usercert.pem` and `~/.globus/userkey.pem`, or they can be bundled in a single file in PKCS12 (Public Key Cryptography Standard \#12), by default named `~/.globus/usercred.p12`. All OSG user tools works with both formats. Unless you specify a name in the command, they look first for the certificate/key pair default names, then for the PKCS12 default name and use the first one they find. PKCS 12 is a single file, convenient to move and used by many softwares, e.g. the Web browsers (so you don't need conversions).
+Your certificate and key pair can be stored as separate files, by default named `~/.globus/usercert.pem` and `~/.globus/userkey.pem`, or they can be bundled in a single file in PKCS12 (Public Key Cryptography Standard \#12), by default named `~/.globus/usercred.p12`. All OSG user tools works with both formats. Unless you specify a name in the command, they look first for the certificate/key pair default names, then for the PKCS12 default name and use the first one they find. PKCS 12 is a single file, convenient to move and used by many applications, e.g. web browsers (so you don't need to convert user certificate format).
```

Comment:
- Let's change PKCS12 into a link to the wikipedia page https://en.wikipedia.org/wiki/PKCS_12. Then get rid of the parenthetical.
- I don't think it's true that all OSG user tools work with both formats. 
- The last sentence is repetitive

Reply:
Done